src/projects/TodoList/components/CreateNewCategory/index.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import './style.css';
+
+class CreateNewCategory extends React.Component {
+  render () {
+    // logic goes here
+
+    let rgb = [];
+    rgb.push(this.props.newCategoryRed, this.props.newCategoryGreen, this.props.newCategoryBlue)
+    let newCategoryColour = {
+      backgroundColor: `rgb(${rgb})`
+    };
+
+    let cancelCreateCategoryButton = this.props.isNewCategoryBeingCreated ? (
+      <button
+        className="btn btn-outline-danger"
+        onClick={this.props.newCategoryBeingCreatedFalse}
+      >
+        Cancel
+      </button>
+    ) : (
+      null
+    )
+
+    let colourPicker = this.props.isNewCategoryBeingCreated ? (
+      <React.Fragment>
+        <div className="row">
+          <div className="col-2 input-colour">
+            Red
+          </div>
+          <input
+            className="col-4 input-range"
+            type="range"
+            min="0" max="255"
```

Comment:
When you're display a component like this over multiple lines, each prop should be on its own line

Reply:
I'll change it up but I think min max on the same line looks decent

Reply:
done